DeployJRE/.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not using the project eliminates the need to import an extra project; as I see it, that can be a significant time saver when you are switching between different local clones of the project.